ajax/libs/angular-ui-tinymce/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "angular-ui-tinymce",
+  "filename": "tinymce.min.js",
+  "version": "0.0.18",
+  "description": "This directive allows you to add a tinymce to your form elements.",
+  "author": "https://github.com/angular-ui/ui-tinymce/graphs/contributors",
+  "license": "MIT",
+  "homepage": "https://angular-ui.github.io/",
+  "keywords": [
+    "tinymce",
+    "AngularJS",
+    "directive"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/angular-ui/ui-tinymce.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/angular-ui/ui-tinymce.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "tinymce*"
```

Comment:
@PeterDaveHello 
I carelessly wrote the files in "dist" identical to those in "src", and I have modified it.
Please review it again. Thanks!